common/build-style/python-module.sh

```diff
@@ -29,6 +29,23 @@ do_build() {
 	done
 }
 
+do_check() {
+	: ${python_versions:="2.7 $py3_ver"}
+
+	for pyver in $python_versions; do
+		ln -s build-${pyver} build
+		if [ -z "$make_check_target" ]; then
+			if ! python${pyver} setup.py --help test >/dev/null 2>&1; then
+				msg_warn "No command 'test' defined by setup.py for python${pyver}.\n"
+				return 0
```

Comment:
won't returning here prevent the build symlink from being deleted ?

Reply:
ah, true. fixed